data/adapters/dbSNFP_adapter.py

```diff
@@ -119,11 +104,27 @@ def long_data(pos):
                     except:
                         return None
 
-                # gene_name + transcript_id + hgvsp + hgvs + splicing (in case pos == -1)
-                key = data(12) + '_' + data(14) + '_' + \
-                    (data(23) or '') + '_' + (data(22) or '')
-
-                if long_data(11) == -1:
+                alt = data(3)
+                if alt == 'X' and 'Ter' in hgvsp:
```

Comment:
X can be in both in aaref and aaalt, will need to convert to * if either of them are X (no need checking if Ter in hgvsp)